.github/actions/setup-java-build/action.yml

```diff
@@ -16,8 +16,8 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: "Cache local Maven repository"
-      uses: actions/cache@v3
+    - name: "Restore cache local Maven repository"
+      uses: actions/cache/restore@704facf57e6136b1bc63b828d79edcd491f0ee84 # v3.3.2
```

Comment:
no one is going to save the cache if you move to restore here

Reply:
also this is the setup-java action: we're not using it in most of our repos: I think this change is not needed for the task at hand